libs/blocks/caas/caas.css

```diff
@@ -424,6 +424,7 @@ dt.title.is-expanded button {
 
 .config-panel dd.content.is-expanded {
   margin-top: 0;
+  max-height: 999999px; /* needed for transition animation */
```

Comment:
Have you explored using `fit-content`? It should be supported by all browsers without a prefix.